sssd_test_framework/roles/client.py

```diff
@@ -70,6 +71,11 @@ def __init__(self, *args, **kwargs) -> None:
         Managing local overrides users and groups.
         """
 
+        self.pam: PAMUtils = PAMUtils(self.host, self.fs)
+        """
+        Managing PAM modules; pam_access and pam_faillock.
```

Comment:
I'd use `Configuring various PAM modules.` to make it more explicit. And remove pam_access and pam_faillock, otherwise we will need to always update this line when we add new modules.

Reply:
It can also go to BaseLinuxRole, I think.